converter_1_test.go

```
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+)
+
+func Test_myStrToInt(t *testing.T){
+	testData := "23"
+	expected , _ := 23 , error(nil)
+    actual , _ := myStrToInt(testData)
```

Please fix align.

---

Why do you ignore an error here ? Error should also be checked in the tests.